spec/controllers/hubstats/deploys_controller_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+module Hubstats
+  RSpec.describe DeploysController, :type => :controller do
+    describe "#create" do
+      it 'should create a deploy' do
+        @deploy = build(:deploy)
```

Comment:
Neither of these are invoking your controller.  The tests need to use your controller to provide the benefit of test coverage.